webroot/src/pages/Login/Login.ts

```diff
@@ -0,0 +1,54 @@
+//
+//  Login.ts
+//  CompactConnect
+//
+//  Created by InspiringApps on 8/12/2024.
+//
+
+import { Component, Vue } from 'vue-facing-decorator';
+
+@Component({
+    name: 'Login',
+    components: {}
+})
+export default class Login extends Vue {
+    //
+    // Lifecycle
+    //
+    created() {
+        this.redirectToHostedLogin();
+    }
+
+    //
+    // Computed
+    //
+    get shouldRemoteLogout(): boolean {
```

Comment:
What's the style guide say about when to list the return type vs not? I've seen you take both approaches and wonder if there are certain parameters that dictate when to take which approach.

Reply:
From earlier comment: We prefer to define the types of computed getters where possible. The places we don't have them yet are when returning stores; we don't have all the types wired up for the store layer yet.